docs/repo-docs/guides/tools/typescript.mdx

```diff
@@ -73,7 +73,9 @@ Inside `packages/typescript-config`, you have a few `json` files which represent
 }
 ```
 
-<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">`tsconfig` options reference</LinkToDocumentation>
+<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">
```

Comment:
Careful, we need this newline so it renders correctly.

Reply:
wait, sorry, I got it backwards.  do you mean you need the newline _not_ to be there?  whatever the case I'm reverting this line to what it was before

Reply:
ok I reverted it to just like it was before this PR (assuming that's correct) 91ef9fca1b9f8adce13bb14cf72ccf711a7555bc